taskchampion/taskchampion/src/server/cloud/gcp.rs

```diff
@@ -241,13 +243,21 @@ mod tests {
     /// Note that the Rust test runner will still show "ok" for the test, as there is no way to
     /// indicate anything else.
     fn make_service() -> Option<(GcpService, impl Fn(&str) -> Vec<u8>)> {
-        let Ok(bucket) = std::env::var("GCP_TEST_BUCKET") else {
-            return None;
+        let bucket = match std::env::var("GCP_TEST_BUCKET") {
+            Ok(val) => val,
+            Err(_) => return None,
+        };
```

Comment:
Why did this change from the `let Ok(..) = .. else { .. }` syntax?

Reply:
Ah I thought this was a bit better in terms of exception handling, I'll change it back immediately